numpy/random/_generator.pyx

```diff
@@ -800,6 +800,11 @@ cdef class Generator:
         ``p`` must sum to 1 when cast to ``float64``. To ensure this, you may wish
         to normalize using ``p = p / np.sum(p, dtype=float)``.
 
+        When passing ``a`` as an integer type (e.g. ``np.int64`` or ``int``) and ``size`` is
+        not specified, the return type is a native Python ``int``. In such a case, ``integers`` 
+        returns an ``np.int64``. However, ``integers`` does not sample non-uniformly (i.e., it
+        does not have a ``p`` parameter) and it only allows sampling with replacement.
```

Comment:
Yeah, I wondered about that. In the end, I thought it would be useful to know somewhere in the docs about how ``integers`` does not allow non-uniform sampling or sampling without replacement (it is orthogonal to the original bug I filed). I could put that info in a separate paragraph or in the ``integers`` documentation as "if you want non-uniform sampling or sampling without replacement, see ``choice``". What do you think?

Reply:
I don't think it's necessary.